src/Surfnet/StepupMiddleware/CommandHandlingBundle/Configuration/Command/CreateInstitutionConfigurationCommand.php

```diff
@@ -19,8 +19,9 @@
 namespace Surfnet\StepupMiddleware\CommandHandlingBundle\Configuration\Command;
 
 use Surfnet\StepupMiddleware\CommandHandlingBundle\Command\AbstractCommand;
+use Surfnet\StepupMiddleware\CommandHandlingBundle\Command\ManagementExecutable;
 
-class CreateInstitutionConfigurationCommand extends AbstractCommand
+class CreateInstitutionConfigurationCommand extends AbstractCommand implements ManagementExecutable
```

Comment:
I see what you mean. 

In our current application, `CreateInstitutionConfigurationCommands` are sent based on events caused by earlier commands. These earlier commands can be sent by any user (i.e. SelfService for `CreateIdentityCommand` and Middleware for `AddToWhitelistCommand`).

In the code, I assume the command ought to be created by Middleware since the incoming events and commands are handled there. I would have to check, but my assumption could be wrong because the earlier and resulting commands all happen within the same transaction, thus would bound to the same user.